src/main/java/de/uni_tuebingen/ub/nppm/db/EinzelbelegDB.java

```diff
@@ -42,6 +42,18 @@ public static Einzelbeleg getFirstPublicEinzelbeleg() throws Exception {
     public static void insertBySql(String sql) throws Exception {
         insertOrUpdate(sql);
     }
+    
+    public static int countEinzelbelegByQuelleId(int qId) throws Exception {
```

Comment:
As mentioned, this function should be removed & implemented within the Quelle model class.

Reply:
i have removed the function